lib/setup.js

```diff
@@ -2293,7 +2293,7 @@ function processCommand(command, args, params, callback) {
                 let oldHostname;
                 let newHostname;
                 if (!change) {
-                    console.warn('Please write "' + tools.appName + ' host this" to use this host ("' + tools.getHostName() + '") in ' + tools.appName + ' for all instances.');
+                    console.warn('Please write "' + tools.appName + ' host this" to use actual hostname ("' + tools.getHostName() + '") in ' + tools.appName + ' for all instances.');
```

Comment:
Klingt nach falschem Englisch. Wie wäre es mit "... to use the current hostname ..."?

Reply:
Ack.